plutus-core/plutus-core/src/PlutusCore/Error.hs

```diff
@@ -111,8 +118,15 @@ data Error uni fun ann
 makeClassyPrisms ''Error
 deriving stock instance (Show fun, Show ann, Closed uni, Everywhere uni Show, GShow uni, Show ParserError) => Show (Error uni fun ann)
 
-instance AsParserError (Error uni fun ann) where
-    _ParserError = _ParseErrorE
+instance AsParserErrorBundle (Error uni fun ann) where
+    _ParserErrorBundle = prism' putError takeError
```

Comment:
I think you should be able to use the same implementation as the others, i.e. `_ParseErrorBundleB`, using the prism generated by `makeClassyPrisms`.

Reply:
I could if `ParseErrorE` carries the wrapper but I'm kinda cheating and trying to carry megaparsec's `ParseErrorBundle` straight. I thought it would be less annoying? I had planned to add a comment.